Specs/DateTime/German/DateTimeModel.json

```diff
@@ -33,8 +33,8 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
-    "Comment": "Currently not supported. 04 will be parsed as the year",
-    "NotSupported": "dotnet, javascript, python",
+    "Comment": "Should be removed, as 22.04 could be an (incorrectly) formatted date (22.04.) or time (22:04).",
```

Comment:
If you saw this sentence written this way? Why interpretation would you tend to choose?

Reply:
I would understands that it is day.year but only because of the "am" and as it is "22". "Ich komme 22.04 zurück" would be ambigous. This could be hh.mm or dd.yy. "Ich komme 12.04 zurück" would be almost impossible as it could be hh.mm or dd.mm or mm.yy. I would decide not to support this case with the missing training "." , at least for now.